paywall/src/utils/isUnlocked.ts

```diff
@@ -1,11 +1,5 @@
 import { keyExpirationTimestampFor } from './keyExpirationTimestampFor'
 import { optimisticUnlocking } from './optimisticUnlocking'
-
-declare let __ENVIRONMENT_VARIABLES__: {
```

Comment:
This cannot work in a module... so we are now passing these values to the function